backend/src/main/java/com/onair/hearit/common/exception/ErrorCode.java

```diff
@@ -9,7 +9,8 @@ public enum ErrorCode {
     INVALID_INPUT(HttpStatus.BAD_REQUEST, "입력값이 유효하지 않습니다."),
     ALREADY_EXIST(HttpStatus.CONFLICT, "이미 존재하는 데이터입니다."),
     NOT_FOUND(HttpStatus.NOT_FOUND, "해당 정보를 찾을 수 없습니다."),
-    UNAUTHORIZED(HttpStatus.UNAUTHORIZED, "인증되지 않은 요청입니다.");
+    UNAUTHORIZED(HttpStatus.UNAUTHORIZED, "인증되지 않은 요청입니다."),
+    INTERNAL_SERVER_ERROR(HttpStatus.INTERNAL_SERVER_ERROR, "서버 내부 오류");
```

Comment:
한 줄 바꿀 때마다 항상 2줄이 Files changed에 뜨기 때문에 아래와 같이 바꿔주면 좋습니다.

```suggestion
INTERNAL_SERVER_ERROR(HttpStatus.INTERNAL_SERVER_ERROR, "서버 내부 오류"),
;
```

Reply:
확실히 enum의 마지막 항목에서도 쉼표를 붙여 Git diff 최소화하는 방법이 좋아보이네요! 수정했습니다!